test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -1032,9 +1052,7 @@ spec setup =
 
             Test.specify "should not replace if Empty term" <|
                 a.replace '' "#" . to_vector . should_equal ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
-                a.replace '' "#" use_regex=True . to_vector . should_equal ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
                 a.replace d "#" . to_vector . should_equal ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
-                a.replace d "#" use_regex=True . to_vector . should_equal ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
```

Comment:
I don't like these tests being removed.

What happens now when I provide an empty regex to replace? I'd like to have this edge tested to ensure no bad behaviour occurs.

Reply:
You cannot create an empty regex, which is tested elsewhere.